code/modules/antagonists/clockcult/clock_structures/ark_of_the_clockwork_justicar.dm

```diff
@@ -162,7 +163,7 @@
 			var/atom/movable/target = L
 			if(isobj(L.loc))
 				target = L.loc
-			target.forceMove(get_turf(pick(GLOB.generic_event_spawns)))
+			target.forceMove(find_safe_turf())
```

Comment:
Note this does have the potential to now randomly teleport players into say, the SME's rad collector areas, or the gravity generator chamber, which would more than likely kill them with radiation. Could also just straight up dump them into the middle of the AI core, AI upload, ect.

Reply:
Pretty sure that people are only chucked out of Reebe when the ark finishes or blows up, and then the round's over.